x-pack/plugins/security_solution/scripts/endpoint/common/vm_services.ts

```diff
@@ -378,3 +375,55 @@ export const getHostVmClient = (
     ? createVagrantHostVmClient(hostname, vagrantFile, log)
     : createMultipassHostVmClient(hostname, log);
 };
+
+/**
+ * Retrieve a list of running VM names
+ * @param type
+ * @param name
+ * @param log
+ */
+export const findVm = async (
+  type: SupportedVmManager,
+  /** Filter results by VM name */
+  name?: string | RegExp,
+  log: ToolingLog = createToolingLogger()
+): Promise<{ data: string[] }> => {
+  log.verbose(`Finding [${type}] VMs with name [${name}]`);
+
+  if (type === 'multipass') {
+    const list = JSON.parse((await execa.command(`multipass list --format json`)).stdout) as {
```

Comment:
previously there was `--format=json`, can these be used interchangeably? Or the previous one was not working correctly? :)